setup.py

```diff
@@ -25,8 +25,12 @@
                       "PyYAML",
                       "pexpect",
                       "werkzeug",
+                      "future",
                       ],
     include_package_data=True,
+    package_data={
+        'marabunta': ['html/*.html'],
+    },
```

Comment:
is it required as soon as we have it in MANIFEST.in?

Reply:
Well without it, they (.html files) were not installed but present in the dist/...tar file
But maybe it has to do with the specific way the lib is installed with my test on camptocamp/docker-odoo-project/pull/41